src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -698,6 +713,16 @@ private static bool IsNodeOrAnyAncestorLeftSideOfDot(SyntaxNode node, ISyntaxFac
 
             return false;
         }
+
+        protected static CustomColumnInfo GetContainingTypeInfo(SyntaxNode node, ISyntaxFactsService syntaxFacts)
+        {
+            return new CustomColumnInfo("ContainingTypeInfo", syntaxFacts.GetNameOfContainingType(node));
+        }
+
+        protected static CustomColumnInfo GetContainingMemberInfo(SyntaxNode node, ISyntaxFactsService syntaxFacts)
+        {
+            return new CustomColumnInfo("ContainingMemberInfo", syntaxFacts.GetNameOfContainingMember(node));
```

Comment:
Should these use the referenced `ColumnDefinition.ColumnName` instead of strings?